buildSrc/src/main/kotlin/buildsrc/convention/kotlin-jvm.gradle.kts

```diff
@@ -0,0 +1,57 @@
+package buildsrc.convention
+
+import Deps
+import org.jetbrains.kotlin.gradle.dsl.ExplicitApiMode
+import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
+
+plugins {
+    `java-library`
+    kotlin("jvm")
+    id("org.jetbrains.dokka")
+
+    id("buildsrc.convention.base")
+    id("buildsrc.convention.detekt")
+}
+
+dependencies {
+    testImplementation(Deps.jUnit)
+    testImplementation(Deps.strikt)
+    testImplementation(Deps.Mockk.mockk)
+    testImplementation(Deps.Mockk.dslJvm)
+}
```

Comment:
Just a comment: normally I prefer keeping dependencies out of convention plugins. buildSrc doesn't play very nicely with the dependency management plugins and tools (like version catalogs), and it can make the subprojects config a bit more obfuscated. But this works fine, especially for common test dependencies, and because `Deps.kt` is well managed and organised.